src/chum.cpp

```diff
@@ -31,7 +32,7 @@ Chum::Chum(QObject *parent)
 {
   connect(&m_ssu, &Ssu::updated, this, &Chum::repositoriesListUpdated);
   connect(&m_ssu, &Ssu::updated, this, &Chum::repoUpdated);
-  connect(Daemon::global(), &Daemon::updatesChanged, [this]() { this->getUpdates(); });
+  connect(Daemon::global(), &Daemon::updatesChanged, this, [this]() { this->getUpdates(); });
 
```

Comment:
As we have lambda functions, I think that it is way clearer to use the original syntax and not add `this` to `connect`.

Reply:
The check is this one https://github.com/KDE/clazy/blob/master/docs/checks/README-connect-3arg-lambda.md.  The this object is the context parameter which ensures the lambda isnt executed if the object is deleted.  Im no expert, and merely relying on the clazy devs knowledge here.

Reply:
Interesting point, then it does make sense